.github/workflows/check-nix-format.yml

```
@@ -36,10 +36,9 @@ jobs:
           echo "targetRev=$targetRev" >> "$GITHUB_ENV"
           echo "target=$target" >> "$GITHUB_ENV"
 
-      - name: Get Nixpkgs revision for nixfmt
+      - name: Get Nixpkgs revision for keep-sorted
```

Was this change unintended?

```suggestion
      - name: Get Nixpkgs revision for nixfmt
```

---

Ah. You modified the wrong file.

---

Fixed it, sorry.